### PR TITLE
Update job templates and README

### PR DESCRIPTION
**Pull request by YuanTingHsieh**
### Description

Follow up PR of re-factoring PR https://github.com/NVIDIA/NVFlare/pull/2164

1. Add `Pipe` explanation in ml-to-fl/np/README.md

2. Two major changes to configs:
    - The "launch_once" argument is moved to "SubprocessLauncher", modify configs and READMEs accordingly
    - The "root_path" argument of "FilePipe" is changed from an empty string to "{WORKSPACE}/{JOB_ID}/{SITE_NAME}" follows PR https://github.com/NVIDIA/NVFlare/pull/2145, this is more clear to the users

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: job_templates/sag_nemo/app_1/config_fed_client.conf [hunk 1]
```diff
@@ -65,14 +59,19 @@
...
-      # This "launcher" component
+      # component id is "launcher"
       id = "launcher"
 
-      # the class path of the component
+      # the class path of this component
       path = "nvflare.app_common.launchers.subprocess_launcher.SubprocessLauncher"
 
-      # the launcher will invoke the script
-      args.script = "python3 -m torch.distributed.run {ddp_config} custom/{app_script}  {app_config} "
```
Comment (holgerroth) [inline job_templates/sag_nemo/app_1/config_fed_client.conf:75]: Shouldn't this remain a distributed run? We need to support multi-gpu training for nemo.

Comment (YuanTingHsieh) [reply job_templates/sag_nemo/app_1/config_fed_client.conf:75]: yes, it should, I copy and paste and accidentally change this, thanks for catching this

File: examples/hello-world/step-by-step/cifar10/cse/cse.ipynb [hunk 2]
```diff
@@ -27,7 +27,7 @@
     "We will be using the [Client API FL code](../code/fl/train.py) trainer converted from the original [Training a Classifer](https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html) example.\n",
     "\n",
     "Key changes when writing a FL code to support multiple tasks:\n",
-    "- When using the default `launch_once` Client API Launcher Executor parameter, we encapsulate our code in `while flare.is_running():` loop so we can call `flare.receive()` and perform various tasks. This is useful when launching everytime would be inefficient, such as when having to perform data setup every time.\n",
+    "- When using the default `launch_once` parameter of "SubprocessLauncher", we encapsulate our code in `while flare.is_running():` loop so we can call `flare.receive()` and perform various tasks. This is useful when launching everytime would be inefficient, such as when having to perform data setup every time.\n",
```
Comment (SYangster) [inline examples/hello-world/step-by-step/cifar10/cse/cse.ipynb:30]: The "SubprocessLauncher" quotes might be causing an "invalid json" rendering issue

Comment (YuanTingHsieh) [reply examples/hello-world/step-by-step/cifar10/cse/cse.ipynb:30]: OH, that's why these colors, I was wondering why,  let me remove those quotes
Thanks

**Pull request merged**